src/routes/load/container/Load.jsx

```diff
@@ -37,8 +38,13 @@ class Load extends React.Component<Props> {
   onLoadSafeSubmit = async (values: Object) => {
     try {
       const { addSafe } = this.props
+      const web3 = getWeb3()
       const safeName = values[FIELD_LOAD_NAME]
-      const safeAddress = values[FIELD_LOAD_ADDRESS]
+      const safeAddressNotChecksum = values[FIELD_LOAD_ADDRESS]
+      let safeAddress = safeAddressNotChecksum
```

Comment:
why do we need this `safeAddressNotChecksum` var? couldn't it just be
```
let safeAddress = values[FIELD_LOAD_ADDRESS]
if (safeAddress) {
  safeAddress = web3.utils.toChecksumAddress(safeAddress)
}
```

Reply:
Yes that makes more sense, I updated it now